Migrate from AAD Pod Identity to Workload Identity (Supported by User Managed Identities) & Update AKS Version (#326)
mspnp/aks-baseline: 04-networking.md

   The spoke creation will emit the following:

     * `appGwPublicIpAddress` - The Public IP address of the Azure Application Gateway (WAF) that will receive traffic for your workload.

>      * `clusterVnetResourceId` - The resource ID of the Virtual network that the cluster, App Gateway, and related will be deployed in. E.g. `/subscriptions/[id]/resourceGroups/rg-enterprise-networking-spokes/providers/Microsoft.Network/virtualNetworks/vnet-spoke-BU0001A0008-00`

Reviewer: ```suggestion
     * `clusterVnetResourceId` - The resource ID of the Virtual network where the cluster, App Gateway, and related will be deployed. E.g. `/subscriptions/[id]/resourceGroups/rg-enterprise-networking-spokes/providers/Microsoft.Network/virtualNetworks/vnet-spoke-BU0001A0008-00`
```

Author: Good reword, update will be coming shortly.

Author: Resolved in: https://github.com/mspnp/aks-baseline/pull/326/commits/3f3058684316dffe84b7b6180ccbf5a212c27a94

>      * `clusterVnetResourceId` - The resource ID of the Virtual network that the cluster, App Gateway, and related will be deployed in. E.g. `/subscriptions/[id]/resourceGroups/rg-enterprise-networking-spokes/providers/Microsoft.Network/virtualNetworks/vnet-spoke-BU0001A0008-00`

Reviewer 2: Are you missing a noun after related? The related what will be deployed?

Author: Good catch, update will be coming shortly.

Author: Resolved in: https://github.com/mspnp/aks-baseline/pull/326/commits/3f3058684316dffe84b7b6180ccbf5a212c27a94
